### DISCUSSION TITLE
Can I run docker containers on UDM Pro SE ?

### DISCUSSION THREAD

**Discussion by KarlBaumann** [unifi-utilities/unifi-common#359] [Q&A]
Sorry for another noob question: I see that everyone is talking about running podman containers, but what about docker?
There is one awesome container I really need to run: https://github.com/keshavdv/unifi-cam-proxy/
It allows using 3rd party cameras with Unifi Protect.

**Comment by boostchicken** [comment]
I am not sure.  Does it have podman installed?
